SampleCode/Net/azure-sdk-samples-for-net/Samples/Sample_AnalyzeWithPrebuiltModel.cs

```diff
@@ -0,0 +1,76 @@
+﻿// coding: utf - 8
+// --------------------------------------------------------------------------
+// Copyright(c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+// --------------------------------------------------------------------------
+
+// Symbol to instruct that the example analyzes local document file.
+// If want to analyze the document file by url, just update below line to: #undef ANALYZE_LOCAL_FILE
```

Comment:
Add some URL to reference the full schema for each prebuilt model

Reply:
Fixed thx